src/test.cpp

```diff
@@ -1,12 +1,12 @@
 // test.cpp
-#include <sqlite3.h>
-#include <stdio.h>
 #include <fmt/core.h>
+#include <sqlite3.h>
+
+#include <cstdio>
```

Comment:
do we need `cstdio`?

Reply:
thanks I've removed this header.

Reply:
i still see it there, but no problem, it could be removed later